src/main/java/com/lineate/mdzinarishvili/reminderapp/models/Reminder.java

```diff
@@ -1,5 +1,94 @@
 package com.lineate.mdzinarishvili.reminderapp.models;
 
+import com.lineate.mdzinarishvili.reminderapp.enums.RecurrenceType;
+
+import java.sql.Date;
+import java.sql.Time;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.Arrays;
+
+
 public class Reminder {
-    
+    private Long id;
+    private String title;
+    private RecurrenceType recurrence;

```

Comment:
Use LocalDateTime

Reply:
I was looking at the requirements and the user should have a time zone. Would that require a different datatype like timestamp? Or is it just for the user to have it linked with their profile, and changing it doesn't impact the time of the reminders?

Reply:
Use DateTime in java, And UTC timestamp in the db